seeder/src/main/interfaces/express_urls.ts

```diff
@@ -1,4 +1,4 @@
-import { get_free_port } from "$utils/port";
+import { get_free_port } from "../utils/port";
```

Comment:
see tsc cant compile from declaration files with alias? lol

Reply:
> see tsc cant compile from declaration files with alias? lol


we either skip type checks, or fix it. I cant allow this type of code at scale

Reply:
> > see tsc cant compile from declaration files with alias? lol
> 
> we either skip type checks, or fix it. I cant allow this type of code at scale

yea but we cant avoid checks either ig,
what bout adding its type on that file itself? instead of dynamic typing?
better IMO

Reply:
>  what bout adding its type on that file itself? instead of dynamic typing?

This looks tsconfig error

Reply:
This is a known issue

- https://github.com/TypeStrong/ts-node/discussions/1450#discussion-3563207